docs/src/pages/getting-started/templates/dashboard/Dashboard.tsx

```diff
@@ -94,7 +95,9 @@ const useStyles = makeStyles((theme) => ({
   appBarSpacer: theme.mixins.toolbar,
 }));
 
-export default function Dashboard() {
+const defaultTheme = createTheme();
```

Comment:
Sometimes, we call it `theme` other `defaultTheme`.

Reply:
I think I renamed it to `defaultTheme` where we have `theme` in the inner context, for example in the component. Should I name it everywhere `defaultTheme`?

Reply:
Ok, all good then